cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -242,6 +242,62 @@ def from_unitary(u: np.ndarray) -> Optional['SingleQubitCliffordGate']:
     def transform(self, pauli: Pauli) -> PauliTransform:
         return self._rotation_map[pauli]
 
+    def to_phased_xz_gate(self) -> phased_x_z_gate.PhasedXZGate:
+        """Convert this gate to a PhasedXZGate instance.
+
+        The rotation can be categorized by {axis} * {degree}:
+            * Identity: I
+            * {x, y, z} * {90, 180, 270}  --- {X, Y, Z} + 6 Quarter turn gates
+            * {+/-xy, +/-yz, +/-zx} * 180  --- 6 Hadamard-like gates
+            * {middle point of xyz in 4 Quadrant} * {120, 240} --- swapping axis
+        note 1 + 9 + 6 + 8 = 24 in total.
+
+        To associate with Clifford Tableau, it can also be grouped by 4:
+            * {I,X,Y,Z} is [[1 0], [0, 1]]
+            * {+/- X_sqrt, 2 Hadamard-like gates acting on the XZ plane} is [[1, 0], [1, 1]]
```

Comment:
```suggestion
            * {+/- X_sqrt, 2 Hadamard-like gates acting on the XY plane} is [[1, 0], [1, 1]]
```

Reply:
Sorry, I found both this one and the comment in the following Z_sqrt was wrong as well. Fixed now. (Comments under the if-else branches are correct).